ci/test_python.sh

```diff
@@ -3,10 +3,6 @@
 
 set -euo pipefail
 
-# TODO: Enable dask query planning (by default) once some bugs are fixed.
-# xref: https://github.com/rapidsai/cudf/issues/15027
-export DASK_DATAFRAME__QUERY_PLANNING=False
-
```

Comment:
AIUI this is one of the OPS relevant changes. Basically removing a workaround that is no longer needed